docs/interpolation.rst

```diff
@@ -1,22 +1,203 @@
+=============================================    ==================  ============    ==================================================================================================
+:any:`GridDataInterpolator`                      Interpolation       Arbitrary       See also :any:`scipy.interpolate.griddata`.
+:any:`ParametrizedNearestSimplexExtrapolator`    Extrapolation       1D or 2D        Linear (1D) or baryzentric (2D) extension outside the grid's convex hull from the nearest simplex.
+:any:`ParametrizedNearestNeighborSearcher`       Nearest Neighbor    Arbitrary       Nearest neighbor finder usable instead of inter- and/or extrapolation. 
+=============================================    ==================  ============    ==================================================================================================
 
-Reference/API
+For components represented by discretized PDFs (PSF and EDISP tables) these classes are:
+
+=============================================    ==================  ============    ==============================================================================
+**Name**                                         **Type**            **Grid-Dim**    **Note**
+=============================================    ==================  ============    ==============================================================================
+:any:`QuantileInterpolator`                      Interpolation       Arbitrary       Adaption of [Hol+13]_ and [Rea99]_ to discretized PDFs.
+:any:`MomentMorphInterpolator`                   Interpolation       1D or 2D        Adaption of [Baa+15]_ to discretized PDFs.
+:any:`MomentMorphNearestSimplexExtrapolator`     Extrapolation       1D or 2D        Extension of [Baa+15]_ beyond the grid's convex hull from the nearest simplex.
+:any:`DiscretePDFNearestNeighborSearcher`        Nearest Neighbor    Arbitrary       Nearest neighbor finder usable instead of inter- and/or extrapolation. 
+=============================================    ==================  ============    ==============================================================================
+
+.. [Hol+13] B. E. Hollister and A. T. Pang (2013). Interpolation of Non-Gaussian Probability Distributions for Ensemble Visualization.
+    https://engineering.ucsc.edu/sites/default/files/technical-reports/UCSC-SOE-13-13.pdf
+.. [Rea99] A. L. Read (1999). Linear Interpolation of Histograms.
+    Nucl. Instrum. Methods Phys. Res. A 425, 357-360. https://doi.org/10.1016/S0168-9002(98)01347-3
+.. [Baa+15] M. Baak, S. Gadatsch, R. Harrington and W. Verkerke (2015). Interpolation between 
+    multi-dimensional histograms using a new non-linear moment morphing method
+    Nucl. Instrum. Methods Phys. Res. A 771, 39-48. https://doi.org/10.1016/j.nima.2014.10.033
+
+
+Usage Example
 -------------
 
+To create a estimator class for an IRF component not yet implemented, one can simply 
```

Comment:
The usage examples should start with the most common use case, i.e. just interpolating a new IRF from a set of grid point IRFs. 

Developing a new method is advanced and I'd leave that for developer documentation, not as the first example for the *user* documentation.